src/SharedFrameworkValidation/SharedFrameworkValidation.proj

```diff
@@ -10,7 +10,6 @@
     <LKGSharedFrameworkExtractPath>$(RefRootPath)sharedFrameworkZip.LKG/shared/Microsoft.NETCore.App/$(_LKGSharedFrameworkVersion)</LKGSharedFrameworkExtractPath>
     <_RunTestsScriptName Condition="'$(RunningOnUnix)' != 'true'">CloneAndRunTests.cmd</_RunTestsScriptName>
     <_RunTestsScriptName Condition="'$(RunningOnUnix)' == 'true'">CloneAndRunTests.sh</_RunTestsScriptName>
-    <_PublishRID>$(RuntimeOS)-$(ArchGroup)</_PublishRID>
```

Comment:
All I would change is the definition of _PublishRID here to use TargetRID instead, but all of the other changes on this file and the RestoreSDKProject should be undone.